### ISSUE TITLE
Add option to DataLakeCatalog/DataCatalogDatabase that crawlers will use existing table

### ISSUE THREAD

**Issue by karnik** [awslabs/data-solutions-framework-on-aws#490]
DataLakeCatalog/DataCatalogDatabase should have the option of manually setting the tables for the crawler as parameters. There are several use cases that require a manually created catalog table.

- You want to choose the catalog table name manually and not rely on the catalog table naming algorithm
- Reuse the table later in the stack (e.g., in a lambda to query the table)

**See:** https://docs.aws.amazon.com/glue/latest/dg/tables-described.html#update-manual-tables

**Comment by vgkowski** [comment]
Good point! My suggestion is we add a crawler mode parameter with 3 different values:
- Can create tables, update schemas and add partitions
- Can only update schemas and add partitions
- Can only add partitions 
What do you think?

**Comment by karnik** [comment]
Sounds great. How do you want to pass the table definition?

**Comment by vgkowski** [comment]
Do you want to be able to filter tables by S3 prefix?
